drafting/conda-template.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+# This script reads in a yaml file (qiime2 recipe)
+# Pulls the needed data from that file
+# And uses that data to write to a secondary yaml file
+# Which will be the end result: a conda recipe
+
+import yaml
+import os
+import sys
+
+if __name__ == '__main__':
+    # Reading the qiime2 recipe file
+    qiime2_recipe = sys.argv[1]
+    if sys.argv[1] == None:
+        raise ValueError('Missing required parameter. QIIME 2 recipe filepath must be provided.')
```

Comment:
I think there is still a bit of a logic error here - have you tried running the script without any arguments?

```bash
$ ./conda-template.py
Traceback (most recent call last):
  File "./conda-template.py", line 14, in <module>
    qiime2_recipe = sys.argv[1]
IndexError: list index out of range
```

Reply:
@thermokarst thank you, you're absolutely right - I changed the logic for these blocks, and everything works as intended now!

Reply:
It looks like you might've reverted a bit too far and somehow lost your ValueErrors - can you please put those back in?